composer.json

```diff
@@ -83,6 +99,7 @@
         "drupal/migrate_upgrade": "^3.2",
         "drupal/moderation_sidebar": "^1.4",
         "drupal/noreferrer": "^1.10",
+        "drupal/password_policy": "^3.0",
```

Comment:
So we took this out of NIDirect because it caused some problems (can't recall what, specifically) and replaced with a more basic, custom code module (nidirect_password_policy). Perhaps we can convert that module into an origins one that can be re-used across any site we operate?